doc/source/examining/loading_data.rst

```diff
@@ -3082,3 +3082,33 @@ non-default cosmological parameters, you may pass an empty dictionary.
    import yt
 
    ds = yt.load(filename, cosmology_parameters={})
+
+.. _loading-cfradial-data:
+
+CfRadial Data
+-------------
+
+yt provides support for loading cartesian-gridded CfRadial netcdf-4 files. When
```

Comment:
@zssherman want to add a sentence or adjust the wording here to describe what a CfRadial dataset is?

Reply:
oops, meant to do a multi-line select there... you'll wanna open up the full file to read the full description.